plexpy/webserve.py

```diff
@@ -101,6 +102,7 @@ def welcome(self, **kwargs):
             return serve_template(templatename="welcome.html", title="Welcome", config=config)
 
     @cherrypy.expose
+    @addtoapi()
     def discover(self, token=''):
```

Comment:
Needs to check for a token before writing to the config. I accidentally wiped my token and couldn't figure out why stuff stopped working.